contracts/src/utils/structs/checkpoints/generic_size.rs

```diff
@@ -0,0 +1,89 @@
+//! Contains generic size utilities for checkpoint storage contract.
+
+use core::ops::{Add, Div, Mul, Sub};
+
+use alloy_primitives::Uint;
+use stylus_sdk::prelude::*;
```

Comment:
Import only what we need.

Reply:
What's the purpose of `prelude` module in the crate then? It should be meant to auto export all basic things. To not spell them all over again. If auto exported name collides with one of declared within your module, then export them explicitly.
Otherwise I see usage of `::prelude::*` more helpful and less polluting the code-base.
@ggonzalez94 wdt?